packages/fauna/.prettierrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+    semi: true,
+    trailingComma: 'all',
+    singleQuote: true,
+    printWidth: 120,
+    tabWidth: 4,
+};
```

Comment:
I wonder if you'd be ok to go with

```suggestion
module.exports = {
    semi: false,
    trailingComma: 'es5',
    singleQuote: true,
    printWidth: 80,
    tabWidth: 2,
};
```

This would keep your changes much closer to the original. The main reasons are

1. the files used no semicolons so far, so the diff would be easier to review
2. same for trailing commas (though this is minor and I don't mind either way)
3. prettier [strongly recommends a print width of 80](https://prettier.io/docs/en/options.html#print-width), which I find very understandable
4. changing the tab width also changed all the JSON files. And `npm i` will change the indentation of the package.json, which is annoying to format back and forth.

That would make it possible for me to actually take a look at the contents of your changes ;)